fastlane/lanes/build.rb

```diff
@@ -175,6 +175,13 @@
       dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
     )
 
+    upload_gutenberg_sourcemaps(
+      sentry_project_slug: SENTRY_PROJECT_SLUG_WORDPRESS,
+      release_version: release_version_current,
+      build_version: build_code_current,
+      app_identifier: WORDPRESS_BUNDLE_IDENTIFIER
+    )
```

Comment:
This source map will be used for the App Store builds (beta and production) of the **WordPress app**.